<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {
	border-image: url(:/.all/src/BG2.png);
}

#btn_start {
	
	border-image: url(:/-1_aiDialog/src/aiDialog1_O.png);
}

#btn_start:hover {
	
	border-image: url(:/-1_aiDialog/src/aiDialog1_P.png);
}

#btn_start:pressed {
	
	border-image: url(:/-1_aiDialog/src/aiDialog1_H.png);
}

#btn_start:checked {
	
	border-image: url(:/-1_aiDialog/src/aiDialog1_H.png);
}

#btn_quit {
	
	border-image: url(:/-1_aiDialog/src/aiDialog2_O.png);
}

#btn_quit:hover {
	
	border-image: url(:/-1_aiDialog/src/aiDialog2_H.png);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>HY그래픽M</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>버튼을 눌러 주문하세요.

예시) 카페라떼 한잔 줘</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_start">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_quit">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_InExact">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="inputStr_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>입력 결과: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ai_InExactListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_ReStart">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>다시 입력하기</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>다음으로</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_Exact">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="menuName_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>menuName_</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="menuImg_">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
           <property name="text">
            <string>menuImg_</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="menuDesc_">
           <property name="text">
            <string>menuDesc_</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QListWidget" name="aiOptionListWidget">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string>옵션 선택</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="itemPrice_"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_back">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>이전으로</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>장바구니에 추가</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../UI/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_start</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>btn_Start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>1044</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_back</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>btn_Back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>620</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>btn_SelectOption()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>-83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>btn_addCart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>807</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ReStart</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>btn_Back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>-53</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_quit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>Close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>-45</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_Start()</slot>
  <slot>btn_Back()</slot>
  <slot>btn_SelectOption()</slot>
  <slot>btn_addCart()</slot>
  <slot>Close()</slot>
 </slots>
</ui>
